web/src/Mainpage.tsx

```diff
@@ -0,0 +1,142 @@
+import { Swiper, SwiperSlide } from "swiper/react";
+import { Navigation, Pagination, Scrollbar, A11y } from "swiper/modules";
+import { useState } from "react";
+import JuiceComp from "./component/JuiceComp.tsx";
+
+function Mainpage() {
```

Comment:
보통 한 파일에 한개의 함수를 export 할 때는
```suggestion
export default function Mainpage() {}
```
이렇게 function과 export default를 한번에써